6-plot_recon_error_kappa.R

```diff
@@ -98,26 +110,24 @@ error.master.df$perc.seq <- factor(error.master.df$perc.seq,
                                    levels = seq(0, 100, by = 10))
 
 # get norm and reconstruction methods as factors 
-error.master.df$normalization <- as.factor(error.master.df$normalization)
-error.master.df$error.measure <- as.factor(error.master.df$error.measure)
+error.master.df$norm.method <- as.factor(error.master.df$norm.method)
 
 # rename platforms -- same as above for kappa data.frame
 error.master.df$platform <- car::recode(error.master.df$platform,
                                         recodes = plt.recode.str)  
 error.master.df$platform <- as.factor(error.master.df$platform)
 
 # reconstruction method as factor
-error.master.df$method <- as.factor(error.master.df$method)
+error.master.df$comp.method <- as.factor(error.master.df$comp.method)
 
 # take the average of each genes error across replicates 
 error.mean.df <- error.master.df %>% 
-                    group_by(gene, perc.seq, normalization, error.measure, 
-                             platform, method) %>% 
+                    group_by(gene, perc.seq, norm.method, comp.method, 
```

Comment:
`dplyr::group_by`? Although I don't actually think its absolutely essential in cases where the namespace is obvious

Reply:
This is a good point. I explicitly call `library(dplyr)` in this particular script and I don't do things like `ggplot2::ggplot()`, so I think I'm going to leave it like this to be consistent. Something to keep in mind in the future, though.